docker/couchbase-tools/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM couchbase:enterprise-6.0.3
```

Comment:
should we use enterprise or community?

Reply:
@tdmanv community image doesn't have cbbackupmgr.